src/server/system_services/pool_server.js

```diff
@@ -560,11 +583,20 @@ async function scale_hosts_pool_job(pool, system) {
         // Scale up (add more storage agents)
         try {
             await pool_ctrl.scale(configured_host_count);
+            const timeout = calc_scale_up_timeout(host_count_diff);
+            await system_store.make_changes({
+                update: {
+                    pools: [{
+                        _id: pool._id,
+                        'hosts_pool_info.init_timeout': Date.now() + timeout
+                    }]
+                }
+            });
```

Comment:
Why do u need this here, the pool is not in initialization any more it's in scaling. unless you want a scaling failed status (which I didn't see in the changes)

Reply:
we must reset the timeout to go back to initializing state. I'll consider adding a check if we're in not initialized before